scrape/manager.go

```diff
@@ -184,6 +185,20 @@ func (m *Manager) reload() {
 	}
 	m.mtxScrape.Unlock()
 	wg.Wait()
+
+	activeTargets := make(map[uint64]*Target)
+	for _, scrapePool := range m.scrapePools {
+		for _, target := range scrapePool.activeTargets {
+			if t, ok := activeTargets[target.labels.Hash()]; ok {
+				level.Warn(m.logger).Log(
+					"msg", "Found targets with same labels after relabelling",
```

Comment:
I wonder if this should print the full set of labels?  (`target.labels.String()`)